www/dusv03/Firma.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="cs">
+
+<head>
+    <title>Humanic#2</title>
+    <meta charset="UTF-8">
+</head>
+
+<body>
+    <header>
+        <span style="color:crimson">Humanic#2 inc.</span>
+        <nav>
+            <ul>
+                <li><a href="index.html">Produkty</a></li>
+                <li><a href="contact.html">Výroba</a></li>
+                <li><a href="Zivotopis.html">Software</a></li>
+                <li><a href="Zivotopis.html">Servis</a></li>
+                <li><a href="Zivotopis.html">Naše zařízení</a></li>
+                <li><a href="Zivotopis.html">Kontakt</a></li>
+                <li><a href="Zivotopis.html">O nás</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <h1>Humanic#2 inc.</h1>
+
+    <h3>Kdo jsme ?</h3>
+    <p> Naše společnost se zabývá pěstováním biobotů(nebo-li robotů s tělem biologického původu) pro veškeré účely.
+        Sežeňte si dokonalého partnera
+        nebo kvalifikovanou pracovní sílu do na firmy. Díky rozsáhlím možnostem cutomizace si u nás vybere každý.
+    </p>
+
+    <Form>
```

Comment:
Proc je tady Form a jeste s velkym "F"?

dej tam div

Reply:
takže budou dva vnořený divy ?

Reply:
Jo, to je validni